### ISSUE TITLE
Visual Studio "Previous IPDB not found, fall back to full compilation."

### ISSUE THREAD

**Issue by Codinablack** [premake/premake-core#2204]
_Before opening a new issue, please [use the issue search feature](https://github.com/premake/premake-core/issues) to see if a similar issue already exists. If not, please help us help you by filling in the template below._

**What are you trying to do?**
I am trying to generate a visual studio project solution, that when used, can properly find the "previous ipdb". 

**What problem are you having?**
The project solution when ran never finds the previously compiled ipdb, and therefore does a completely full compilation again.

**What have you tried so far?**
I have tried changing debugformat values, as well as various flags and options.
I have also tried directing the visual studio solution through premake to the location of the ipdb.

**What version of Premake are you using?**
I am using Dev Branch, compiled on my own.

**Anything else we should know?**
I have tried changing settings manually in the visual studio project without luck.

Here is my premake script.
[premake5.txt](https://github.com/premake/premake-core/files/14986339/premake5.txt)

**Comment by Jarod42** [comment]
>
```
filter "system:windows"
    openmp "On"
    characterset "MBCS"
    debugformat "c7"
    flags {"NoIncrementalLink"}
    linkoptions {"/IGNORE:4099"}
filter {}
```
(Better to place premake5.lua content in block code instead of link BTW)

isn't `debugformat "c7"` problematic?

**Comment by Codinablack** [comment]
> > 
> 
> ```
> filter "system:windows"
>     openmp "On"
>     characterset "MBCS"
>     debugformat "c7"
>     flags {"NoIncrementalLink"}
>     linkoptions {"/IGNORE:4099"}
> filter {}
> ```
> 
> (Better to place premake5.lua content in block code instead of link BTW)
> 
> isn't `debugformat "c7"` problematic?

I don't think so, but the reason debugformat "c7" was even added in the first place was to try to solve this issue, like I stated above, I have tried different values, but also only even found debugformat because I was trying to solve this problem.

**Comment by Codinablack** [comment]
Anyone have any clue how I can resolve this??
